### DISCUSSION TITLE
4A0CF1BE-3A52-4D3A-B33D-C9C798C8F830

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#202] [General]
# 4A0CF1BE-3A52-4D3A-B33D-C9C798C8F830

Zeta Andromedae is a binary star system located in the constellation of Andromeda. It's approximately 189 light-years away from our Solar system and the system consists of a K-type giant and a smaller late-G/early-K type main-sequence star.

This addon adds a fictional but in regard to the real data of the star Zeta Andromedae, specifically the primary star which is a K-type giant mentioned earlier.

https://celestia.mobi/addon?item=4A0CF1BE-3A52-4D3A-B33D-C9C798C8F830

**Comment by exocritic5457** [comment]
Exactly the thing I want to see. Textures based on real data is such a sterling niche, and this is a hill I will die on.
